args_test.go

```diff
@@ -160,6 +163,9 @@ func (t *TestSuite) Test_binArgs_parse(c *C) {
 	c.Check(args.Parent, Equals, true)
 	c.Check(args.Passthru, Equals, true)
 	c.Check(args.Sensitive, Equals, true)
+	c.Check(args.Tags, HasLen, 2)
```

Comment:
Can you please switch this to an `assert` so the test fails before we'd hit the out of bounds error?